src/common/src/types/mod.rs

```diff
@@ -122,7 +122,7 @@ pub enum DataType {
     #[display("date")]
     #[from_str(regex = "(?i)^date$")]
     Date,
-    #[display("varchar")]
+    #[display("character varying")]
     #[from_str(regex = "(?i)^varchar$")]
```

Comment:
Where do we depend on this? 👀

Reply:
Are you asking what program relies on this format type? If so, then here it it [type deserialization](https://github.com/sqlalchemy/sqlalchemy/blob/72f37f535979c86a45c8a0b381a0ae636a8f2e8a/lib/sqlalchemy/dialects/postgresql/base.py#L1609) of SqlAlchemy.
Hum, if you're asking if there are any program that already rely on this `varchar` format type, I don't know 🤕.

Reply:
I've confirmed that it's only used in tests (like `expr::build_from_pretty`), so it's not a big deal. However, what about also changing this for consistency? Take `int32` as an example of how to accept both:

https://github.com/risingwavelabs/risingwave/blob/01c7c2bed08c447cca9a927e626d95498ce271b8/src/common/src/types/mod.rs#L108

Reply:
Is this what you mean:
```rust
#[from_str(regex = "(?i)^character varying$|^varchar$")]
```

Reply:
committed it, plz review.